tests/test_barostat.py

```diff
@@ -136,14 +139,19 @@ def test_barostat_is_deterministic():
     seed = 2021
     np.random.seed(seed)
 
-    box_vol = 27.08816
+    box_vol = 26.89966
 
     pressure = 1.0 * unit.atmosphere
 
     mol_a = hif2a_ligand_pair.mol_a
     ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
-    # margin = 0 to match original reference value
-    unbound_potentials, sys_params, masses, coords, complex_box = get_solvent_phase_system(mol_a, ff, margin=0.0)
+
+    host_system, host_coords, host_box, host_top = build_water_system(3.0)
```

Comment:
Is there are reason to do this over `get_solvent_phase_system`?

Reply:
Don't want to minimize, its dramatically slows down this test.